### ISSUE TITLE
Calista Corporation (NC5) breaks the IEM web app due to coordinates

### ISSUE THREAD

**Issue by BobTorgerson** [ua-snap/data-api#140]
Fix? Remove?

**Comment by charparr** [comment]
Aleut Corp. will break it too. I don't know that any of these are needed or useful for IEM web app. These polygons are more zones, and don't actually indicate land ownership / status. 

![nc](https://user-images.githubusercontent.com/7774271/160899005-41fd0d76-6b70-48ea-98f0-0cd7268ce6c5.png)

**Comment by charparr** [comment]
Yep, still broken! My takes to truncate these polys to "this side" of the antemeridian somewhere around -179.0 degrees longitude and see if that solves the problem. I don't think that reduces any level of usefulness as data cookie cutters. The latitude minimum for Aleut Corp might need to be validated also.

**Comment by charparr** [comment]
This polygon renders in NCR, but no data is displayed. It just has our nice "sorry, no data available for this place" message. The polygon no longer breaks the app.
